lib/livebook_web/live/layout_helpers.ex

```diff
@@ -15,18 +15,20 @@ defmodule LivebookWeb.LayoutHelpers do
 
     ~H"""
     <div class="flex grow h-full">
-      <.live_region role="alert" />
-      <.sidebar
-        socket={@socket}
-        current_page={@current_page}
-        current_user={@current_user}
-        saved_hubs={@saved_hubs}
-      />
+      <div class="absolute sm:static h-full z-[600] sm:z-[500]">
```

Comment:
I don't think z-index matters once the element is `static`, right?

```suggestion
      <div class="absolute sm:static h-full z-[600]">
```